batch/Makefile

```diff
@@ -70,4 +70,8 @@ deploy: push
 	sed -e "s,@sha@,$$(git rev-parse --short=12 HEAD)," \
 	  -e "s,@image@,$$(cat batch-image)," \
 	  < deployment.yaml.in > deployment.yaml
+	kubectl delete namespace test
```

Comment:
This will crash if namespace test doesn't exist. I think prefacing with `-` to ignore may make sense, since you probably want to be able to deploy regardless of whether the outdated namespace exists?

Reply:
hmm. `test` should definitely always exist, but I guess it doesn't really matter if it's missing, we're about to recreate it.